### PR TITLE
src: kw sendemail support WIP

### PR DESCRIPTION
**Pull request by kwy95**
WIP implementation of `kw send-email`

### CODE REVIEW

**Comment by magalilemes** [comment]
I noticed you're using 'kw s' as the short option for 'kw send-email', but we're already using 'kw s' for 'kw ssh'.

**Comment by kwy95** [comment]
> I noticed you're using 'kw s' as the short option for 'kw send-email', but we're already using 'kw s' for 'kw ssh'.

@magalilemes It's actually `kw se` for now, but I'm open to suggestions for better names. I have also considered `kw mail`, probably without a short version as most of the obvious options are already being used.

**Review by rodrigosiqueira — CHANGES_REQUESTED** [review]
Hi @kwy95 ,

I did not review this PR in detail because I noticed that this is just something to start a discussion. I suggest that you close this PR and create a new one that has this:

1. Commit 1: Add basic support for git send-email
 * Add: kw mail --setup --smtpserver "" --smtpuser ""
 * Add tests and documentation

2. Commit 2: Add support to verify
 * Add: kw mail --setup --verify
 * Add test and doc

3. Commit 3: Add support to interactive setup
 * Add: kw mail --setup -i
 * Add tests and doc

That's it for your first version. After this new PR gets merged, you should focus on --send.

File: src/kw_config_loader.sh [hunk 1]
```diff
@@ -20,21 +20,19 @@ declare -gA deploy_target_opt=(['vm']=1 ['local']=2 ['remote']=3)
...
   if [[ "$1" =~ -h|--help ]]; then
     vars_help "$1"
     exit 0
   fi
 
   if [ -f "$PWD/kworkflow.config" ]; then
-    has_local_config_path="Yes"
-  else
-    has_local_config_path="No"
+    has_local_config_path='Yes'
```
[the remark this answers is not in the document]
Comment (rodrigosiqueira) [reply src/kw_config_loader.sh:31]: If you want, send this tiny fix in another PR. It does not make sense to add this in this PR.

**Comment by rodrigosiqueira** [comment]
@kwy95 ,
I'm going to close this PR since you have a new one at:
https://github.com/kworkflow/kworkflow/pull/437/commits/e1946d8a3eaab70b6a36aadb950d1941f62b19be

**Pull request closed without merging**